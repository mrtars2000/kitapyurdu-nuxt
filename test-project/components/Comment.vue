<template>
  <v-app>
    <v-card>
      <v-card class="py-5 px-10" style="background-color: #f5faf0;">
        <v-row md="2">
          <v-col md="3">
            <v-card>
              <v-card-title class="d-flex justify-center">
                Ürüne Puan Ver
              </v-card-title>
              <v-card-text>
                <v-rating

                  color="yellow darken-3"
                  background-color="grey darken-1"
                  empty-icon="$ratingFull"
                  half-increments
                  hover
                  large
                  class="d-flex justify-center"
                />

                <p class="d-flex justify-center my-3">
                  İyi
                </p>
              </v-card-text>
            </v-card>
          </v-col>
          <v-col md="8">
            <v-textarea

              solo
              counter
              maxlength="120"
              style=" background-color: white;"
              placeholder="Yorum Yaz"
            />
          </v-col>
          <v-col md="1">
            <v-btn style="margin-top: 140px; background-color: orange;">
              <p class="mb-auto" style="color: white;">
                Gönder
              </p>
            </v-btn>
          </v-col>
        </v-row>
      </v-card>
      <v-card class="py-5 px-10">
        <v-row md="1">
          <v-col md="1" class="my-5">
            <p class="my-auto">
              Sırala:
            </p>
          </v-col>
          <v-col md="3">
            <v-select
              :items="items"
              label="Outlined style"
              outlined
              class="my-1"
              :menu-props="{bottom:true,offsetY:true}"
            />
          </v-col>
          <v-col md="3">
            <v-checkbox

              :label="`Satın Almaları Göster:`"
            />
          </v-col>
          <v-col offset-md="2" md="3" class="my-5">
            <a href="#" style="text-decoration: none; color: #dadada;">Tüm Yorumları Göster</a>
          </v-col>
        </v-row>
      </v-card>
      <v-card class="py-5 px-10">
        <v-row md="8">
          <v-row>
            <v-col md="1">
              <div style="display: inline-block;">
                <v-btn
                  class="mx-2 my-5"
                  fab

                  large
                  color="#ffedc1"
                >
                  <v-icon x-large style="color: #ff8000;">
                    mdi-account
                  </v-icon>
                </v-btn>
              </div>
            </v-col>

            <v-col md="11">
              <div style="display: inline-block;">
                <v-card>
                  <v-card-title>
                    <a style="font-size: small;">
                      cy
                    </a>
                    <v-rating
                      v-model="rating1"
                      background-color="orange lighten-3"
                      color="orange"
                    />
                    <p style="font-size: small;" class="my-1">
                      19 Ekim 2021
                    </p>
                    <v-icon class="mx-2" style="color: #ff8000;">
                      mdi-cart
                    </v-icon>
                  </v-card-title>
                  <v-card-text>
                    <p>
                      Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                    </p>
                  </v-card-text>
                </v-card>
              </div>
            </v-col>
          </v-row>
          <v-row>
            <v-col md="1">
              <div style="display: inline-block;">
                <v-btn
                  class="mx-2 my-5"
                  fab

                  large
                  color="#ffedc1"
                >
                  <v-icon x-large style="color: #ff8000;">
                    mdi-account
                  </v-icon>
                </v-btn>
              </div>
            </v-col>

            <v-col md="11">
              <div style="display: inline-block;">
                <v-card>
                  <v-card-title>
                    <a style="font-size: small;">
                      cy
                    </a>
                    <v-rating
                      v-model="rating1"
                      background-color="orange lighten-3"
                      color="orange"
                    />
                    <p style="font-size: small;" class="my-1">
                      19 Ekim 2021
                    </p>
                    <v-icon class="mx-2" style="color: #ff8000;">
                      mdi-cart
                    </v-icon>
                  </v-card-title>
                  <v-card-text>
                    <p>
                      Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                    </p>
                  </v-card-text>
                </v-card>
              </div>
            </v-col>
          </v-row>
          <v-row>
            <v-col md="1">
              <div style="display: inline-block;">
                <v-btn
                  class="mx-2 my-5"
                  fab

                  large
                  color="#ffedc1"
                >
                  <v-icon x-large style="color: #ff8000;">
                    mdi-account
                  </v-icon>
                </v-btn>
              </div>
            </v-col>

            <v-col md="11">
              <div style="display: inline-block;">
                <v-card>
                  <v-card-title>
                    <a style="font-size: small;">
                      cy
                    </a>
                    <v-rating
                      v-model="rating1"
                      background-color="orange lighten-3"
                      color="orange"
                    />
                    <p style="font-size: small;" class="my-1">
                      19 Ekim 2021
                    </p>
                    <v-icon class="mx-2" style="color: #ff8000;">
                      mdi-cart
                    </v-icon>
                  </v-card-title>
                  <v-card-text>
                    <p>
                      Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                    </p>
                  </v-card-text>
                </v-card>
              </div>
            </v-col>
          </v-row>
          <v-row>
            <v-col md="1">
              <div style="display: inline-block;">
                <v-btn
                  class="mx-2 my-5"
                  fab

                  large
                  color="#ffedc1"
                >
                  <v-icon x-large style="color: #ff8000;">
                    mdi-account
                  </v-icon>
                </v-btn>
              </div>
            </v-col>

            <v-col md="11">
              <div style="display: inline-block;">
                <v-card>
                  <v-card-title>
                    <a style="font-size: small;">
                      cy
                    </a>
                    <v-rating
                      v-model="rating1"
                      background-color="orange lighten-3"
                      color="orange"
                    />
                    <p style="font-size: small;" class="my-1">
                      19 Ekim 2021
                    </p>
                    <v-icon class="mx-2" style="color: #ff8000;">
                      mdi-cart
                    </v-icon>
                  </v-card-title>
                  <v-card-text>
                    <p>
                      Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                    </p>
                  </v-card-text>
                </v-card>
              </div>
            </v-col>
          </v-row>
          <v-row>
            <v-col md="1">
              <div style="display: inline-block;">
                <v-btn
                  class="mx-2 my-5"
                  fab

                  large
                  color="#ffedc1"
                >
                  <v-icon x-large style="color: #ff8000;">
                    mdi-account
                  </v-icon>
                </v-btn>
              </div>
            </v-col>

            <v-col md="11">
              <div style="display: inline-block;">
                <v-card>
                  <v-card-title>
                    <a style="font-size: small;">
                      cy
                    </a>
                    <v-rating
                      v-model="rating1"
                      background-color="orange lighten-3"
                      color="orange"
                    />
                    <p style="font-size: small;" class="my-1">
                      19 Ekim 2021
                    </p>
                    <v-icon class="mx-2" style="color: #ff8000;">
                      mdi-cart
                    </v-icon>
                  </v-card-title>
                  <v-card-text>
                    <p>
                      Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                    </p>
                  </v-card-text>
                </v-card>
              </div>
            </v-col>
          </v-row>
        </v-row>
      </v-card>
      <v-card class="py-5 px-10">
        <v-row md="1">
          <p class="my-2">
            Gösterilen: 1 ile 5 arası, toplam: 675 (135 Sayfa)
          </p>
          <v-spacer />
          <v-pagination
            v-model="page"
            :length="15"
            :total-visible="7"
            next-icon="mdi-menu-right"
          />
        </v-row>
      </v-card>
    </v-card>
  </v-app>
</template>

<script>

export default {
  data () {
    return {
      items: ['Foo', 'Bar', 'Fizz', 'Buzz'],
      rating1: 5,
      page: 1
    }
  }
}
</script>
