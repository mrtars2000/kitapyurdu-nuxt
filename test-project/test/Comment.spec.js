import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import Vuex from 'vuex'
import Comment from '@/components/Comment'
Vue.use(Vuetify)
Vue.use(Vuex)

const factory = () => {
  return shallowMount(Comment, {})
}

describe('Comment.vue', () => {
  // eslint-disable-next-line no-unused-vars
  let vuetify
  let store

  beforeEach(() => {
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    vuetify = new Vuetify()

    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    store = new Vuex.Store({
      data () {
        return {
          items: ['Foo', 'Bar', 'Fizz', 'Buzz'],
          rating1: 5,
          page: 1
        }
      }
    })
  })
  const wrapper = factory()

  test('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })
  // With jest we can create snapshot files of the HTML output
  test('should have a custom title and match snapshot ', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
  test('has a button', () => {
    expect(wrapper.find('.v-btn').exists())
  })
  // We could also verify this differently by checking the text content
  test('We could also verify this differently by checking the text content', () => {
    const wrapper = factory()
    expect(wrapper.find('.v-card-title').text()).toBe('Ürüne Puan Ver')
  })
})
