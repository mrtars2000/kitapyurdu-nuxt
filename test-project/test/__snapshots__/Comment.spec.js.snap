// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment.vue should have a custom title and match snapshot  1`] = `
"<v-app-stub id=\\"app\\">
  <v-card-stub loaderheight=\\"4\\" tag=\\"div\\">
    <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" class=\\"py-5 px-10\\" style=\\"background-color: rgb(245, 250, 240);\\">
      <v-row-stub tag=\\"div\\" md=\\"2\\">
        <v-col-stub md=\\"3\\" tag=\\"div\\">
          <v-card-stub loaderheight=\\"4\\" tag=\\"div\\">
            <v-card-title-stub tag=\\"div\\" class=\\"d-flex justify-center\\">
              Ürüne Puan Ver
            </v-card-title-stub>
            <v-card-text-stub tag=\\"div\\">
              <v-rating-stub color=\\"yellow darken-3\\" opendelay=\\"0\\" closedelay=\\"0\\" ripple=\\"true\\" large=\\"true\\" backgroundcolor=\\"grey darken-1\\" emptyicon=\\"$ratingFull\\" fullicon=\\"$ratingFull\\" halficon=\\"$ratingHalf\\" halfincrements=\\"true\\" hover=\\"true\\" length=\\"5\\" value=\\"0\\" iconlabel=\\"$vuetify.rating.ariaLabel.icon\\" class=\\"d-flex justify-center\\"></v-rating-stub>
              <p class=\\"d-flex justify-center my-3\\">
                İyi
              </p>
            </v-card-text-stub>
          </v-card-stub>
        </v-col-stub>
        <v-col-stub md=\\"8\\" tag=\\"div\\">
          <v-textarea-stub errorcount=\\"1\\" errormessages=\\"\\" messages=\\"\\" rules=\\"\\" successmessages=\\"\\" backgroundcolor=\\"\\" loaderheight=\\"2\\" clearicon=\\"$clear\\" counter=\\"true\\" placeholder=\\"Yorum Yaz\\" solo=\\"true\\" type=\\"text\\" rowheight=\\"24\\" rows=\\"5\\" maxlength=\\"120\\" style=\\"background-color: white;\\"></v-textarea-stub>
        </v-col-stub>
        <v-col-stub md=\\"1\\" tag=\\"div\\">
          <v-btn-stub tag=\\"button\\" activeclass=\\"\\" type=\\"button\\" style=\\"margin-top: 140px; background-color: orange;\\">
            <p class=\\"mb-auto\\" style=\\"color: white;\\">
              Gönder
            </p>
          </v-btn-stub>
        </v-col-stub>
      </v-row-stub>
    </v-card-stub>
    <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" class=\\"py-5 px-10\\">
      <v-row-stub tag=\\"div\\" md=\\"1\\">
        <v-col-stub md=\\"1\\" tag=\\"div\\" class=\\"my-5\\">
          <p class=\\"my-auto\\">
            Sırala:
          </p>
        </v-col-stub>
        <v-col-stub md=\\"3\\" tag=\\"div\\">
          <v-select-stub errorcount=\\"1\\" errormessages=\\"\\" messages=\\"\\" rules=\\"\\" successmessages=\\"\\" appendicon=\\"$dropdown\\" backgroundcolor=\\"\\" label=\\"Outlined style\\" loaderheight=\\"2\\" clearicon=\\"$clear\\" outlined=\\"true\\" type=\\"text\\" valuecomparator=\\"[Function]\\" nodatatext=\\"$vuetify.noDataText\\" items=\\"Foo,Bar,Fizz,Buzz\\" itemcolor=\\"primary\\" itemdisabled=\\"disabled\\" itemtext=\\"text\\" itemvalue=\\"value\\" menuprops=\\"[object Object]\\" class=\\"my-1\\"></v-select-stub>
        </v-col-stub>
        <v-col-stub md=\\"3\\" tag=\\"div\\">
          <v-checkbox-stub errorcount=\\"1\\" errormessages=\\"\\" messages=\\"\\" rules=\\"\\" successmessages=\\"\\" backgroundcolor=\\"\\" label=\\"Satın Almaları Göster:\\" ripple=\\"true\\" valuecomparator=\\"[Function]\\" indeterminateicon=\\"$checkboxIndeterminate\\" officon=\\"$checkboxOff\\" onicon=\\"$checkboxOn\\"></v-checkbox-stub>
        </v-col-stub>
        <v-col-stub md=\\"3\\" offsetmd=\\"2\\" tag=\\"div\\" class=\\"my-5\\"><a href=\\"#\\" style=\\"text-decoration: none; color: rgb(218, 218, 218);\\">Tüm Yorumları Göster</a></v-col-stub>
      </v-row-stub>
    </v-card-stub>
    <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" class=\\"py-5 px-10\\">
      <v-row-stub tag=\\"div\\" md=\\"8\\">
        <v-row-stub tag=\\"div\\">
          <v-col-stub md=\\"1\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-btn-stub color=\\"#ffedc1\\" tag=\\"button\\" activeclass=\\"\\" large=\\"true\\" fab=\\"true\\" type=\\"button\\" class=\\"mx-2 my-5\\">
                <v-icon-stub xlarge=\\"\\" x-large=\\"\\" style=\\"color: rgb(255, 128, 0);\\">
                  mdi-account
                </v-icon-stub>
              </v-btn-stub>
            </div>
          </v-col-stub>
          <v-col-stub md=\\"11\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-card-stub loaderheight=\\"4\\" tag=\\"div\\">
                <v-card-title-stub tag=\\"div\\"><a style=\\"font-size: small;\\">
                    cy
                  </a>
                  <v-rating-stub color=\\"orange\\" opendelay=\\"0\\" closedelay=\\"0\\" ripple=\\"true\\" backgroundcolor=\\"orange lighten-3\\" emptyicon=\\"$ratingEmpty\\" fullicon=\\"$ratingFull\\" halficon=\\"$ratingHalf\\" length=\\"5\\" value=\\"5\\" iconlabel=\\"$vuetify.rating.ariaLabel.icon\\"></v-rating-stub>
                  <p class=\\"my-1\\" style=\\"font-size: small;\\">
                    19 Ekim 2021
                  </p>
                  <v-icon-stub class=\\"mx-2\\" style=\\"color: rgb(255, 128, 0);\\">
                    mdi-cart
                  </v-icon-stub>
                </v-card-title-stub>
                <v-card-text-stub tag=\\"div\\">
                  <p>
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                  </p>
                </v-card-text-stub>
              </v-card-stub>
            </div>
          </v-col-stub>
        </v-row-stub>
        <v-row-stub tag=\\"div\\">
          <v-col-stub md=\\"1\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-btn-stub color=\\"#ffedc1\\" tag=\\"button\\" activeclass=\\"\\" large=\\"true\\" fab=\\"true\\" type=\\"button\\" class=\\"mx-2 my-5\\">
                <v-icon-stub xlarge=\\"\\" x-large=\\"\\" style=\\"color: rgb(255, 128, 0);\\">
                  mdi-account
                </v-icon-stub>
              </v-btn-stub>
            </div>
          </v-col-stub>
          <v-col-stub md=\\"11\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-card-stub loaderheight=\\"4\\" tag=\\"div\\">
                <v-card-title-stub tag=\\"div\\"><a style=\\"font-size: small;\\">
                    cy
                  </a>
                  <v-rating-stub color=\\"orange\\" opendelay=\\"0\\" closedelay=\\"0\\" ripple=\\"true\\" backgroundcolor=\\"orange lighten-3\\" emptyicon=\\"$ratingEmpty\\" fullicon=\\"$ratingFull\\" halficon=\\"$ratingHalf\\" length=\\"5\\" value=\\"5\\" iconlabel=\\"$vuetify.rating.ariaLabel.icon\\"></v-rating-stub>
                  <p class=\\"my-1\\" style=\\"font-size: small;\\">
                    19 Ekim 2021
                  </p>
                  <v-icon-stub class=\\"mx-2\\" style=\\"color: rgb(255, 128, 0);\\">
                    mdi-cart
                  </v-icon-stub>
                </v-card-title-stub>
                <v-card-text-stub tag=\\"div\\">
                  <p>
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                  </p>
                </v-card-text-stub>
              </v-card-stub>
            </div>
          </v-col-stub>
        </v-row-stub>
        <v-row-stub tag=\\"div\\">
          <v-col-stub md=\\"1\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-btn-stub color=\\"#ffedc1\\" tag=\\"button\\" activeclass=\\"\\" large=\\"true\\" fab=\\"true\\" type=\\"button\\" class=\\"mx-2 my-5\\">
                <v-icon-stub xlarge=\\"\\" x-large=\\"\\" style=\\"color: rgb(255, 128, 0);\\">
                  mdi-account
                </v-icon-stub>
              </v-btn-stub>
            </div>
          </v-col-stub>
          <v-col-stub md=\\"11\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-card-stub loaderheight=\\"4\\" tag=\\"div\\">
                <v-card-title-stub tag=\\"div\\"><a style=\\"font-size: small;\\">
                    cy
                  </a>
                  <v-rating-stub color=\\"orange\\" opendelay=\\"0\\" closedelay=\\"0\\" ripple=\\"true\\" backgroundcolor=\\"orange lighten-3\\" emptyicon=\\"$ratingEmpty\\" fullicon=\\"$ratingFull\\" halficon=\\"$ratingHalf\\" length=\\"5\\" value=\\"5\\" iconlabel=\\"$vuetify.rating.ariaLabel.icon\\"></v-rating-stub>
                  <p class=\\"my-1\\" style=\\"font-size: small;\\">
                    19 Ekim 2021
                  </p>
                  <v-icon-stub class=\\"mx-2\\" style=\\"color: rgb(255, 128, 0);\\">
                    mdi-cart
                  </v-icon-stub>
                </v-card-title-stub>
                <v-card-text-stub tag=\\"div\\">
                  <p>
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                  </p>
                </v-card-text-stub>
              </v-card-stub>
            </div>
          </v-col-stub>
        </v-row-stub>
        <v-row-stub tag=\\"div\\">
          <v-col-stub md=\\"1\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-btn-stub color=\\"#ffedc1\\" tag=\\"button\\" activeclass=\\"\\" large=\\"true\\" fab=\\"true\\" type=\\"button\\" class=\\"mx-2 my-5\\">
                <v-icon-stub xlarge=\\"\\" x-large=\\"\\" style=\\"color: rgb(255, 128, 0);\\">
                  mdi-account
                </v-icon-stub>
              </v-btn-stub>
            </div>
          </v-col-stub>
          <v-col-stub md=\\"11\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-card-stub loaderheight=\\"4\\" tag=\\"div\\">
                <v-card-title-stub tag=\\"div\\"><a style=\\"font-size: small;\\">
                    cy
                  </a>
                  <v-rating-stub color=\\"orange\\" opendelay=\\"0\\" closedelay=\\"0\\" ripple=\\"true\\" backgroundcolor=\\"orange lighten-3\\" emptyicon=\\"$ratingEmpty\\" fullicon=\\"$ratingFull\\" halficon=\\"$ratingHalf\\" length=\\"5\\" value=\\"5\\" iconlabel=\\"$vuetify.rating.ariaLabel.icon\\"></v-rating-stub>
                  <p class=\\"my-1\\" style=\\"font-size: small;\\">
                    19 Ekim 2021
                  </p>
                  <v-icon-stub class=\\"mx-2\\" style=\\"color: rgb(255, 128, 0);\\">
                    mdi-cart
                  </v-icon-stub>
                </v-card-title-stub>
                <v-card-text-stub tag=\\"div\\">
                  <p>
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                  </p>
                </v-card-text-stub>
              </v-card-stub>
            </div>
          </v-col-stub>
        </v-row-stub>
        <v-row-stub tag=\\"div\\">
          <v-col-stub md=\\"1\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-btn-stub color=\\"#ffedc1\\" tag=\\"button\\" activeclass=\\"\\" large=\\"true\\" fab=\\"true\\" type=\\"button\\" class=\\"mx-2 my-5\\">
                <v-icon-stub xlarge=\\"\\" x-large=\\"\\" style=\\"color: rgb(255, 128, 0);\\">
                  mdi-account
                </v-icon-stub>
              </v-btn-stub>
            </div>
          </v-col-stub>
          <v-col-stub md=\\"11\\" tag=\\"div\\">
            <div style=\\"display: inline-block;\\">
              <v-card-stub loaderheight=\\"4\\" tag=\\"div\\">
                <v-card-title-stub tag=\\"div\\"><a style=\\"font-size: small;\\">
                    cy
                  </a>
                  <v-rating-stub color=\\"orange\\" opendelay=\\"0\\" closedelay=\\"0\\" ripple=\\"true\\" backgroundcolor=\\"orange lighten-3\\" emptyicon=\\"$ratingEmpty\\" fullicon=\\"$ratingFull\\" halficon=\\"$ratingHalf\\" length=\\"5\\" value=\\"5\\" iconlabel=\\"$vuetify.rating.ariaLabel.icon\\"></v-rating-stub>
                  <p class=\\"my-1\\" style=\\"font-size: small;\\">
                    19 Ekim 2021
                  </p>
                  <v-icon-stub class=\\"mx-2\\" style=\\"color: rgb(255, 128, 0);\\">
                    mdi-cart
                  </v-icon-stub>
                </v-card-title-stub>
                <v-card-text-stub tag=\\"div\\">
                  <p>
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda impedit voluptate praesentium nihil expedita, excepturi cupiditate magnam, aperiam consequuntur, quam omnis. Officiis eius rem itaque odit iusto, facilis a quam?
                  </p>
                </v-card-text-stub>
              </v-card-stub>
            </div>
          </v-col-stub>
        </v-row-stub>
      </v-row-stub>
    </v-card-stub>
    <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" class=\\"py-5 px-10\\">
      <v-row-stub tag=\\"div\\" md=\\"1\\">
        <p class=\\"my-2\\">
          Gösterilen: 1 ile 5 arası, toplam: 675 (135 Sayfa)
        </p>
        <v-spacer-stub tag=\\"div\\"></v-spacer-stub>
        <v-pagination-stub length=\\"15\\" nexticon=\\"mdi-menu-right\\" previcon=\\"$prev\\" totalvisible=\\"7\\" value=\\"1\\" pagearialabel=\\"$vuetify.pagination.ariaLabel.page\\" currentpagearialabel=\\"$vuetify.pagination.ariaLabel.currentPage\\" previousarialabel=\\"$vuetify.pagination.ariaLabel.previous\\" nextarialabel=\\"$vuetify.pagination.ariaLabel.next\\" wrapperarialabel=\\"$vuetify.pagination.ariaLabel.wrapper\\"></v-pagination-stub>
      </v-row-stub>
    </v-card-stub>
  </v-card-stub>
</v-app-stub>"
`;
